### ISSUE TITLE
Gesture for opening menu

### ISSUE THREAD

**Issue by BabyOilJohnson** [Myzel394/NumberHub#49]
Can you add swipe right gesture for hamburger menu opening. It will be convenient.

**Comment by BabyOilJohnson** [comment]
@KjmT6459 can you show me where. I didn't see.

**Comment by KjmT6459** [comment]
It's always been there.
Put your finger to the left of the one and then swipe up towards the middle of ÷ and x. That angle, to nearly straight up, opens it from any starting point on the left side of the screen. 
You can also do it going down but, personally I find that pretty finicky.

**Comment by BabyOilJohnson** [comment]
Yeah, I know that trick. Thanks for that, but I was talking about adding a gesture to swipe to the right to open menu from anywhere, instead of swiping from the left edge.
